native/src/modules/endpoint/sendTrackingSignal.js

```diff
@@ -5,38 +5,58 @@ import AppSettings from '../settings/AppSettings'
 import moment from 'moment'
 import type { SettingsType } from '../settings/AppSettings'
 import buildConfig from '../app/constants/buildConfig'
+import ErrorCodes, { fromError } from '../error/ErrorCodes'
+import type { SignalType } from 'api-client'
 
-const sendTrackingSignal = async ({ signal, offline = false }: {| signal: SpecificSignalType, offline?: boolean |}) => {
+export const sendCompleteSignal = async (signal: SignalType) => {
```

Comment:
Not sure what this name means, are there incomplete signals? Mabye `sendRequest` would be better and below just continue to use `sendTrackingSignal` instead of `sendSpecificSignal`